debian/changelog

```diff
@@ -1,3 +1,16 @@
+authd (0.5.0) questing; urgency=medium
```

Comment:
As per the release document, I'd keep this as `UNRELEASED` now, then when doing the final `git commit -m "Upload to $(dpkg-parsechangelog -SDistribution)"` commit change this (and we can do it as direct push to main at that point, together with the tag).

Reply:
I've followed the release document, which contains:

> 2. Create a separate commit which updates to changelog to target the latest Ubuntu release instead of UNRELEASED

In the **Prepare release branch** section.